src/components/Icons/IconTemplate.jsx

```diff
@@ -0,0 +1,216 @@
+export default function IconTemplate() {
```

Comment:
I created a component that has the big SVG in in so that the ids are accessible in the document.